lib/healthcheck/worker.go

```diff
@@ -361,15 +361,23 @@ func (w *worker) getThreshold(cfg *healthCheckConfig) uint32 {
 
 func (w *worker) setThresholdReached(ctx context.Context) {
 	const transitionReason = types.TargetHealthTransitionReasonThreshold
+	checkWord := pluralize(w.lastResultCount, "check")
 	if w.lastResultErr == nil {
-		msg := fmt.Sprintf("%d health checks passed", w.lastResultCount)
+		msg := fmt.Sprintf("%d health %v passed", w.lastResultCount, checkWord)
```

Comment:
nit: i'm good with our own helper. I see that we already importing this lib so just curious if some of these helpers are useful
https://pkg.go.dev/github.com/dustin/go-humanize@v1.0.1/english